pkg/cmd/cli/cmd/exporter.go

```diff
@@ -27,6 +28,7 @@ import (
 	deployapi "github.com/openshift/origin/pkg/deploy/api"
 	imageapi "github.com/openshift/origin/pkg/image/api"
 	routeapi "github.com/openshift/origin/pkg/route/api"
+	osacontrollers "github.com/openshift/origin/pkg/serviceaccounts/controllers"
```

Comment:
Create an util package for serviceaccounts and move the helpers in there. We don't import controllers anywhere else in the cli code.